file-format-spec.ts

```diff
@@ -0,0 +1,87 @@
+// This file contains types which specify the speedscope file format.
+
+export namespace FileFormat {
+  export interface File {
+    version: string
+    exporter: 'https://www.speedscope.app'
```

Comment:
I guess it's going to be weird to author JSON files that conform to this format but which are not actually exported by speedscope.

I'm considering having the file format be specified as a JSON schema (http://json-schema.org/). In that case, the `$schema` field could be used as a replacement for `exporter` to unambiguously identify a JSON ball as being a speedscope profile.